scripts/seedUsersAndPosts.js

```diff
@@ -0,0 +1,331 @@
+                zipcode: '02108', // Boston
+                profile: {
+                    username: 'higherfurtherfaster',
+                    firstName: 'Carol',
+                    lastName: 'Danvers',
+                    dob: new Date('1968-04-24'),
+                    bio: 'Higher, further, faster. Former Air Force pilot, happy to help my community.',
+                    profilePicture: null
+                },
+                settings: {
+                    dmsEnabled: true
+                }
+            },
+            {
+                email: 'jennifer.walters@law.com',
+                password: 'Password123!',
+                zipcode: '90028', // Los Angeles
+                profile: {
+                    username: 'superhero_at_law',
+                    firstName: 'Jennifer',
+                    lastName: 'Walters',
+                    dob: new Date('1980-05-12'),
+                    bio: 'Attorney at law. Strong advocate for my neighbors (literally and figuratively).',
+                    profilePicture: null
+                },
+                settings: {
+                    dmsEnabled: true
+                }
+            },
+            {
+                email: 'matt.murdock@nelsonmurdock.com',
+                password: 'Password123!',
+                zipcode: '10036', // Hell\'s Kitchen, Manhattan
+                profile: {
+                    username: 'devils_advocate',
+                    firstName: 'Matt',
+                    lastName: 'Murdock',
+                    dob: new Date('1982-10-16'),
+                    bio: 'I\'m not daredevil.',
+                    profilePicture: null
+                },
+                settings: {
+                    dmsEnabled: true
+                }
+            },
+            {
+                email: 'peter.parker@empire.edu',
+                password: 'Password123!',
+                zipcode: '11375', // Forest Hills, Queens
+                profile: {
+                    username: 'friendly_neighbor',
+                    firstName: 'Peter',
+                    lastName: 'Parker',
+                    dob: new Date('2001-08-10'),
+                    bio: 'Your friendly neighborhood helper! Queens born and raised. With great power comes great responsibility.',
+                    profilePicture: null
+                },
+                settings: {
+                    dmsEnabled: true
+                }
+            }
+        ];
+
+        console.log('Creating users...');
+        const createdUsers = [];
+        
+        for (const user of testUsers) {
+            try {
+                const result = await userData.createUser(user);
+                createdUsers.push({ ...result.user, zipcode: user.zipcode });
+                console.log(`Created user: ${user.profile.username}`);
+            } catch (error) {
+                console.error(`Error creating user ${user.profile.username}:`, error.message);
+            }
+        }
```

Comment:
~ 12+ second parallelizing speedup by for testing :)

```suggestion
        // do this first or else you’ll get multiple of the same messages because async hehe
        await userData.server.calculateOptimalSaltRounds();
        
        await Promise.all(
            testUsers.map(async (user) => {
                try {
                    const result = await userData.createUser(user);
                    createdUsers.push({ ...result.user, zipcode: user.zipcode });
                    console.log(`Created user: ${user.profile.username}`);
                } catch (error) {
                    console.error(`Error creating user ${user.profile.username}:`, error.message);
                }
            })
        )
```